python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -115,39 +192,56 @@ def __call__(self, catalog):
 class ColorAnalysisConfig(Config):
     coaddName = Field(dtype=str, default="deep", doc="Name for coadd")
     flags = ListField(dtype=str, doc="Flags of objects to ignore",
-                      default=["base_SdssCentroid_flag", "base_PixelFlags_flag_saturatedCenter",
-                               "base_PixelFlags_flag_interpolatedCenter", "base_PsfFlux_flag"])
+                      default=["slot_Centroid_flag", "slot_Shape_flag",
+                               "base_PsfFlux_flag", "modelfit_CModel_flag",
+                               "base_PixelFlags_flag_saturated", "base_ClassificationExtendedness_flag"])
     analysis = ConfigField(dtype=AnalysisConfig, doc="Analysis plotting options")
     transforms = ConfigDictField(keytype=str, itemtype=ColorTransform, default={},
                                  doc="Color transformations to analyse")
-    fluxFilter = Field(dtype=str, default="HSC-I", doc="Filter to use for plotting against magnitude")
+    fluxFilter = Field(dtype=str, default="HSC-I", doc=("Filter to use for plotting against magnitude and "
+                                                        "setting star/galaxy classification"))
+    fluxFilterGeneric = Field(dtype=str, default="i", doc=("Filter to use for plotting against magnitude "
+                                                           "and setting star/galaxy classification"))
     srcSchemaMap = DictField(keytype=str, itemtype=str, default=None, optional=True,
                              doc="Mapping between different stack (e.g. HSC vs. LSST) schema names")
     toMilli = Field(dtype=bool, default=True, doc="Print stats in milli units (i.e. mas, mmag)?")
+    doPlotPcaColors = Field(dtype=bool, default=True, doc="Create the Ivezic PCA color offset plots?")
     writeParquetOnly = Field(dtype=bool, default=False,
                              doc="Only write out Parquet tables (i.e. do not produce any plots)?")
     doWriteParquetTables = Field(dtype=bool, default=True,
                                  doc=("Write out Parquet tables (for subsequent interactive analysis)?"
-                                      "\nNOTE: if True but fastparquet package is unavailable, a warning is "
-                                      "issued and table writing is skipped."))
+                                      "\nNOTE: if True but fastparquet package is unavailable, a warning "
+                                      "is issued and table writing is skipped."))
+    plotRanges = DictField(keytype=str, itemtype=float, default=
```

Comment:
I don't think a line ending with default= is standard. You have enough space for: 
```
    plotRanges = DictField(keytype=str, itemtype=float,
                           default={"griX0": -0.6, "griX1": 2.0, "griY0": -0.6, "griY1": 3.0,
                                    "rizX0": -0.4, "rizX1": 3.0, "rizY0": -0.2, "rizY1": 1.5,
                                    "izyX0": -0.5, "izyX1": 1.3, "izyY0": -0.4, "izyY1": 0.8,
                                    "z9yX0": -0.3, "z9yX1": 0.45, "z9yY0": -0.2, "z9yY1": 0.5},
                           doc="Plot Ranges for various color-color combinations")
```